### PR TITLE
tor status fix for unconfigured, add update on settings change; fix i2p out led status

### PR DESCRIPTION
**Pull request by RetroPooh**

### CODE REVIEW

**Comment by RetroPooh** [comment]
Note: there seems lot of code around settings that must be covered with #ifdef RS_USE_I2P_SAM3 (by SAM maintainer, i hope) later.

**Comment by defnax** [comment]
@csoler

File: retroshare-gui/src/gui/statusbar/torstatus.cpp [hunk 1]
```diff
@@ -185,6 +185,11 @@ void TorStatus::getTorStatus()
 					torstatusLabel->setToolTip( text + tr("i2p proxy is not available"));
 				}
 			}
+			else
+			{
+				torstatusLabel->setPixmap(FilesDefs::getPixmapFromQtResourcePath(":/icons/tile_inactive_48.png").scaledToHeight(S,Qt::SmoothTransformation));
+				torstatusLabel->setToolTip(tr("No tor configuration"));
```
Comment (csoler) [inline retroshare-gui/src/gui/statusbar/torstatus.cpp:191]: I assume the message should be "Tor proxy is not properly configured. Please go to Preferences -> Network to fix it."

**Comment by defnax** [comment]
@RetroPooh

**Pull request merged**